### PR TITLE
Description list update to improve the horizontal style

### PR DESCRIPTION
**Pull request by Christopher-O**
## What does this pull request (PR) do? / Que fait cette demande « pull » (PR)?
Change bootstrap width default of <dt> for horizontal from 160px to 20ch in order to allow both WET4 and GCWeb to allow the same amount of characters in the <dt> to remain the same for both themes without any overrides.  Added a new class, .dt-max to allow the longest word in a set of  <dt> to dictate the width without any word break.

## Additional information (optional) / Information additionnelle (optionel)

**General checklist / Liste de contrôle générale**

- [X] Create/update documentation /// Créer/mettre à jour la documentation
- [ ] Build and test PR's code /// Rouler le script de compilation et tester le code de la PR
- [ ] Validate changes against WCAG for accessibility /// Valider les changements avec WCAG pour l'accessibilité
- [ ] Ensure documentation is bilingual /// S'assurer que la documentation soit bilingue

### CODE REVIEW

File: src/other/utility/utility-fr.hbs [hunk 1]
```diff
@@ -535,7 +539,21 @@
...
+<p>Add the class <code>dt-max</code> with the <code>dl-horizontal</code> class and the longest word in any of the term (<code>&lt;dt&gt;</code>) elements will determine the width of all the terms.</p>
+<dl class="dl-horizontal max-dt">
+	<dt>Description title</dt>
+	<dd>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</dd>
+	<dt>Long word: incomprehensibilities</dt>
+	<dd>Etiam at erat et sapien tempus vulputate.</dd>
+	<dt>A fews filling words that take space</dt>
+	<dd>Quisque tristique dui in ante mattis, nec commodo tortor eleifend.</dd>
+</dl>
+<!--Translation ended //-->
```
Comment (duboisp) [inline src/other/utility/utility-fr.hbs:555]:
```suggestion
<p>Si le mot le plus long est en majuscule, certains navigateurs peuvent ne pas afficher de trait d'union.</p>  
<p>Si vous souhaitez définir l'endroit où il y aura une opportunité de créer un trait d'union pour faire un renvoie à la ligne en cas de manque d'espace, vous pouvez utiliser l'entité du trait d'union souple  (<code>&shy;</code>).</p>
<h3>Largeur maximal de la colonne des termes</h3>
<p>Ajouter la classe CSS <code>dt-max</code> avec la classe CSS <code>dl-horizontal</code> afin que le plus long mot dans la colonne des termes (l'élément <code>&lt;dt&gt;</code>) détermine la largeur pour l'ensemble des termes.</p>
<dl class="dl-horizontal max-dt">
	<dt>Titre du terme</dt>
	<dd>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</dd>
	<dt>Long mot: incompréhensibilités</dt>
	<dd>Etiam at erat et sapien tempus vulputate.</dd>
	<dt>Quelques mots de remplissage qui prennent de l'espace</dt>
	<dd>Quisque tristique dui in ante mattis, nec commodo tortor eleifend.</dd>
</dl>
```

**Review by duboisp — CHANGES_REQUESTED** [review]
Proposed some French translation and pointed the trailing space error reported by our CI script.

Note: I didn't tested this PR neither reviewed the propose CSS code change yet.

File: src/other/utility/utility-fr.hbs [hunk 2]
```diff
@@ -525,6 +525,8 @@
 
 <h2 id="dl-horizontal">Liste descriptive horizontale</h2>
 
+<h3>Renvoi à la ligne</h3>
+	
 <p>Ceci est la mise à l'essaie d'un cas d'utilisation spécifique où le mot est tronqué dans le titre de la description lorsque ce dernier est long et prend toute l'espace disponible.</p>
```
Comment (duboisp) [inline src/other/utility/utility-fr.hbs:530]: There is an extra space to remove here:

```suggestion
<h3>Renvoi à la ligne</h3>

<p>Ceci est la mise à l'essaie d'un cas d'utilisation spécifique où le mot est tronqué dans le titre de la description lorsque ce dernier est long et prend toute l'espace disponible.</p>
```

File: src/other/utility/utility-fr.hbs [hunk 3]
```diff
@@ -535,6 +537,18 @@
 	<dt>Quelques mots afin de remplir de l'espace</dt>
 	<dd>Quisque tristique dui in ante mattis, nec commodo tortor eleifend.</dd>
 </dl>
+<p>Si le mot le plus long est en majuscule, certains navigateurs peuvent ne pas afficher de trait d'union.</p>  
+<p>Si vous souhaitez définir l'endroit où il y aura une opportunité de créer un trait d'union pour faire un renvoie à la ligne en cas de manque d'espace, vous pouvez utiliser l'entité du trait d'union souple  (<code>&shy;</code>).</p>
```
Comment (duboisp) [inline src/other/utility/utility-fr.hbs:541]: There is trailing space here
```suggestion
<p>Si le mot le plus long est en majuscule, certains navigateurs peuvent ne pas afficher de trait d'union.</p>
<p>Si vous souhaitez définir l'endroit où il y aura une opportunité de créer un trait d'union pour faire un renvoie à la ligne en cas de manque d'espace, vous pouvez utiliser l'entité du trait d'union souple  (<code>&shy;</code>).</p>
```

**Review by duboisp — DISMISSED** [review]

**Comment by duboisp** [comment]
Pre-approved upon review and local testing.

File: src/base/bootstrap-overrides/_core.scss [hunk 4]
```diff
@@ -66,6 +67,22 @@ a {
 			padding-bottom: 0;
 		}
 	}
+	&.dt-max {
```
Comment (Garneauma) [inline src/base/bootstrap-overrides/_core.scss:70]: This class is not used in the examples. They use `.max-dt`. Please change whichever accordingly.

**Review by Garneauma — CHANGES_REQUESTED** [review]
@Christopher-O Thank you for your contribution. There are only two small changes to do in order for this to be merged.

- [x] Edit class name in either the SCSS or examples
- [x] Fix mobile version

I have tried using the longest word in the English dictionary and the layout broke on mobile. The styles should only be applied SM breakpoint and up to avoid any unwanted layout break. To do so, include your `.dt-max` styles inside the media query block starting on line 41 of _core.scss. (https://github.com/wet-boew/wet-boew/pull/9644/files#diff-c94529051aec32b579486da6a9d91ddd0d9086cfbd7a3a27a55e6e90813fa7f4R41)

File: src/base/bootstrap-overrides/_core.scss (hunk 4 above)
Comment (Christopher-O) [reply src/base/bootstrap-overrides/_core.scss:70]: I have updated the utilities page to dt-max to align with dl-horizontal (front loading the element).  Thank you for catching that.

**Comment by Christopher-O** [comment]
> 

Moved dt-max to inside the mobile media width query.  Thank you again for catching that @Garneauma

**Review by Garneauma — APPROVED** [review]

**Pull request merged**